webdriver-spec.html

```diff
@@ -2261,6 +2273,14 @@
       <p>Otherwise, set the "<code>proxy</code>" entry
        in <var>matched capabilities</var> to <var>capability value</var>.
 
+     <dt>"<code>setWindowRect</code>"
+     <dd><p>If <var>capability value</var> is not a boolean
```

Comment:
I think I'm misunderstanding how this algorithm is used, then. I thought the
effect of this would be an "invalid argument" error if the "New Session" command
was executed with a body like:

```json
{
  "setWindowRect": "I made a mistake and specified a string instead of a boolean"
}
```

...which seems to me like a good way to avoid confusion in ambiguous
circumstances. But based on your feedback, I'm thinking this is *not* the
actual effect of this language. Could you confirm?